libs/block-editor/src/lib/extensions/dot-bubble-menu.extension.ts

```diff
@@ -6,11 +6,11 @@ import { shouldShowBubbleMenu } from '../utils/bubble-menu.utils';
 import { BubbleMenuComponentProps } from '@dotcms/block-editor';
 
 const defaultTippyOptions: Partial<Props> = {
-    appendTo: document.body,
     duration: 500,
     maxWidth: 'none',
     placement: 'top-start',
-    trigger: 'manual'
+    trigger: 'manual',
+    interactive: true
```

Comment:
What this does?

Reply:
`interactive:  true` allows tippy to have interactive content within it so that it can be hovered over and clicked without hiding it.